### DISCUSSION TITLE
Fuzzing on post param and headers

### DISCUSSION THREAD

**Discussion by AlessandroZ** [projectdiscovery/fuzzing-templates#31] [Q&A]
Hello, 

Reading your documentation (https://nuclei.projectdiscovery.io/templating-guide/protocols/http-fuzzing/#part), I have read: 
```
Support will be added for path,header,body,cookie, etc parts soon.
```
Do you have some idea if it will be added soon or I build something on my own. Just to know if it's just planned to add it but could be done in one year or it should be done in few weeks. It could give me an idea if I build my own tool for this specific usage. 

Thank a lot for all your work. You have done an amazing job.

**Comment by ehsandeep** [comment]
@AlessandroZ, we plan to work on it in the coming months; let me know if you have any specific request/use case/suggestion to consider.

**Comment by AlessandroZ** [comment]
Thank you for your answer. 

My only request is when the "header" part are tested, headers on the yaml file should be tested but also ones passed as argument (e.g `nuclei -H "XXXXX" ...` ). 

Thanks a lot for all your work. Have a nice day.
